--- src/ash/output.c
+++ src/ash/output.c
@@ -230,6 +230,7 @@ out1fmt(const char *fmt, ...)
 	va_end(ap);
 }
 
+#if 0
 void
 dprintf(const char *fmt, ...)
 {
@@ -240,6 +241,7 @@ dprintf(const char *fmt, ...)
 	va_end(ap);
 	flushout(out2);
 }
+#endif
 
 void
 fmtstr(char *outbuf, size_t length, const char *fmt, ...)
--- src/ash/output.h
+++ src/ash/output.h
@@ -65,8 +65,10 @@ void outfmt(struct output *, const char *, ...)
     __attribute__((__format__(__printf__,2,3)));
 void out1fmt(const char *, ...)
     __attribute__((__format__(__printf__,1,2)));
+    /*
 void dprintf(const char *, ...)
     __attribute__((__format__(__printf__,1,2)));
+    */
 void fmtstr(char *, size_t, const char *, ...)
     __attribute__((__format__(__printf__,3,4)));
 void doformat(struct output *, const char *, va_list);
--- src/kash/output.c
+++ src/kash/output.c
@@ -226,6 +226,7 @@ out1fmt(shinstance *psh, const char *fmt, ...)
 	va_end(ap);
 }
 
+#if 0
 void
 dprintf(shinstance *psh, const char *fmt, ...)
 {
@@ -236,6 +237,7 @@ dprintf(shinstance *psh, const char *fmt, ...)
 	va_end(ap);
 	flushout(psh->out2);
 }
+#endif
 
 void
 fmtstr(char *outbuf, size_t length, const char *fmt, ...)
--- src/kash/output.h
+++ src/kash/output.h
@@ -69,8 +69,10 @@ void outfmt(struct output *, const char *, ...)
     __attribute__((__format__(__printf__,2,3)));
 void out1fmt(struct shinstance *, const char *, ...)
     __attribute__((__format__(__printf__,2,3)));
+    /*
 void dprintf(struct shinstance *, const char *, ...)
     __attribute__((__format__(__printf__,2,3)));
+    */
 void fmtstr(char *, size_t, const char *, ...)
     __attribute__((__format__(__printf__,3,4)));
 void doformat(struct output *, const char *, va_list);
