diff -Naur kbuild-0.1.9998svn2543.orig/kBuild/env.sh kbuild-0.1.9998svn2543/kBuild/env.sh
--- kbuild-0.1.9998svn2543.orig/kBuild/env.sh	2011-08-25 19:09:21.000000000 +0200
+++ kbuild-0.1.9998svn2543/kBuild/env.sh	2011-10-04 00:41:34.000000000 +0200
@@ -358,7 +358,7 @@
         hppa64|parisc64)
             KBUILD_HOST_ARCH='hppa64'
             ;;
-        arm|armv4l|armv5tel|armv5tejl)
+        arm|armv4l|armv5tel|armv5tejl|armv7l)
             KBUILD_HOST_ARCH='arm'
             ;;
         alpha)
